Applications/Core TLP.md

```diff
@@ -0,0 +1,69 @@
+## Introduction
+
+Currently the Node.js Foundation directly oversees the development of the core platform and associated working groups. The addition of Top Level Projects will predictably increase the workload on the TSC and so it makes sense to transfer these responsibilities to a Top Level Project concerned solely with the development of the platform.
+
+## History & Metrics
+
+Created by Ryan Dahl, now the largest programming ecosystem in the world.
+
+## Scope
+
+The Core TLP will have sole responsibility and discretion over the node.js project in the following areas:
+
+* Setting release dates
+* Release quality standards
+* Technical direction
+* Governance process and practices.
+* Contribution process and practices.
+* Maintaining the list of additional Collaborators.
+* Development process and any coding standards.
+* Mediating technical conflicts between Collaborators and Working Groups.
+* Node.js build and CI infrastructure.
```

Comment:
Nit: Some list items end with periods and some don't. Doesn't really matter either way but should probably be consistent.